### PR TITLE
#2107 Add checkbox unit tests

### PR DESCRIPTION
**Pull request by DavidMi-1**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #2107)
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by konekoya** [comment]
retry manager-e2e

File: ohara-manager/client/src/components/common/Mui/Form/__tests__/Checkbox.test.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    jest.clearAllMocks();
+  });
+
+  it('renders checkbox', () => {
+    const props = setup();
+
+    render(<MuiCheckbox {...props} />);
+  });
+
+  it('renders checkbox content', () => {
```
Comment (konekoya) [inline ohara-manager/client/src/components/common/Mui/Form/__tests__/Checkbox.test.js]:
```suggestion
  it('renders checkbox label', () => {
```

File: ohara-manager/client/src/components/common/Mui/Form/__tests__/Checkbox.test.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    const props = setup();
+    const { getByTestId } = render(<MuiCheckbox {...props} />);
+    const checkbox = getByTestId('checkbox').querySelector(
+      'input[type="checkbox"]',
+    );
+
+    expect(checkbox).toHaveProperty('checked', true);
+  });
+
+  it('renders checkbox checked if the checked is false', () => {
```
Comment (konekoya) [inline ohara-manager/client/src/components/common/Mui/Form/__tests__/Checkbox.test.js]:
```suggestion
  it('should not check the checkbox if the checked is false', () => {
```

**Comment by DavidMi-1** [comment]
> Good one, please address the comments :)

OK, got it. Obviously, I'm very poor at expression ability and English . :)

**Comment by konekoya** [comment]
> OK, got it. Obviously, I'm very poor at expression ability and English . :)

No worries, none of us are native speakers :)

**Pull request merged**